addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -197,43 +200,46 @@
 </template>
 
 <template id="header" name="Forum Index">
-    <t t-if="forum.active" t-call="website_forum.layout">
-        <div class="oe_structure" id="oe_structure_website_forum_header_1"/>
-        <div id="wrap" t-attf-class="o_wforum_wrapper position-relative container d-flex d-xl-grid px-lg-0 #{website_forum_action}">
-            <t t-call="website_forum.user_sidebar"/>
-            <t t-call="website_forum.user_sidebar_mobile"/>
-            <div class="o_wforum_content_wrapper w-100 w-xl-auto px-lg-3">
-                <div class="o_wprofile_email_validation_container row g-0 justify-content-center mb-3 mb-lg-5 pt-2 pt-lg-3">
-                    <t t-call="website_profile.email_validation_banner">
-                        <t t-set="redirect_url" t-value="'/forum/%s' % forum.id"/>
-                        <t t-set="additional_validation_email_message"> and join this Forum</t>
-                        <t t-set="additional_validated_email_message"> You may now participate in our forums.</t>
-                    </t>
-                    <t t-call="website_forum.model_nav"/>
-                    <t t-out="0"/>
+    <t t-call="website_forum.layout">
```

Comment:
This will be trickier to rebase. (You *may* want to do the same as you did this time and progressively apply changes instead...)